elip-0201.mediawiki

```diff
@@ -0,0 +1,36 @@
+<pre>
+  ELIP: 201
```

Comment:
Numbers are normally assigned after the first draft and not chosen by the ELIP author